content/docs/add-react-to-a-website.md

```diff
@@ -1,202 +1,203 @@
+Opcjonalnie: [Pobierz cały przykład (2KB zipped)](https://gist.github.com/gaearon/6668a1f6986742109c00a581ce704605/archive/f6c882b6ae18bde42dcf6fdb751aae93495a2275.zip)
 
-### Step 1: Add a DOM Container to the HTML {#step-1-add-a-dom-container-to-the-html}
+### Krok 1: Dodaj kontener do HTML {#step-1-add-a-dom-container-to-the-html}
 
-First, open the HTML page you want to edit. Add an empty `<div>` tag to mark the spot where you want to display something with React. For example:
+Najpierw otwórz stronę internetowa, która chcesz edytować. Dodaj pusty znacznik '<div>' aby zaznaczyć miejsce w którym chcesz wyświetlić coś za pomocą przycisku React. Na przykład:
 
-```html{3}
-<!-- ... existing HTML ... -->
+'''html{3}
+<!-- ... istniejący kod HTML ... -->
 
 <div id="like_button_container"></div>
 
-<!-- ... existing HTML ... -->
-```
+<!-- ... istniejący kod HTML ... -->
+'''
+
+Przypisaliśmy temu znacznikowi '<div>' unikalny atrybut 'id'. Pozwoli nam to na późniejsze odnalezienie kodu korzystając z JavaScript oraz na wyświetlenie w nim komponentu React.
 
-We gave this `<div>` a unique `id` HTML attribute. This will allow us to find it from the JavaScript code later and display a React component inside of it.
 
->Tip
+>Wskazówka
 >
->You can place a "container" `<div>` like this **anywhere** inside the `<body>` tag. You may have as many independent DOM containers on one page as you need. They are usually empty -- React will replace any existing content inside DOM containers.
+>Możesz umieścić "kontener" '<div>' **w dowolnym miejscu** wewnątrz znacznika '<body>'. Możesz mieć tyle niezależnych kontenerów na jednej stronie, ile potrzebujesz. Zazwyczaj są one puste -- React zastąpi istniejącą zawartość w kontenerze.
 
-### Step 2: Add the Script Tags {#step-2-add-the-script-tags}
+### Krok 2: Dodaj znacznik script {#step-2-add-the-script-tags}
 
-Next, add three `<script>` tags to the HTML page right before the closing `</body>` tag:
+Następnie, dodaj trzy znaczniki '<script>' do strony internetowej tuż przed zamykającym znacznikiem '</body>':
 
-```html{5,6,9}
-  <!-- ... other HTML ... -->
+'''html{5,6,9}
+<!-- ... pozostały kod HTML ... -->
 
-  <!-- Load React. -->
-  <!-- Note: when deploying, replace "development.js" with "production.min.js". -->
-  <script src="https://unpkg.com/react@16/umd/react.development.js" crossorigin></script>
-  <script src="https://unpkg.com/react-dom@16/umd/react-dom.development.js" crossorigin></script>
+<!-- Załaduj React. -->
+<!-- Uwaga: podczas wdrażania, zamień "development.js" na "production.min.js". -->
+<script src="https://unpkg.com/react@16/umd/react.development.js" crossorigin></script>
+<script src="https://unpkg.com/react-dom@16/umd/react-dom.development.js" crossorigin></script>
 
-  <!-- Load our React component. -->
-  <script src="like_button.js"></script>
+<!-- Załaduj nasz komponent React. -->
+<script src="like_button.js"></script>
 
 </body>
-```
+'''
```

Comment:
```suggestion
```
```

Reply:
Nie zmieniaj "back ticków" na apostrofy, bo strona będzie się błędnie wyświetlać.

Reply:
To z automatu mi wpadlo, jak sprawdzalem ogonki itp.